### PR TITLE
Lots of typos!

### PR DESCRIPTION
**Pull request by natolambert**

### CODE REVIEW

File: stackllama.md [hunk 1]
```diff
@@ -266,6 +266,27 @@ Once you add the evaluation step, the fun begins: you can start iterating on you
...
+                     Younes Belkada and
+                     Kashif Rasul and
+                     Lewis Tunstall and
+                     Leandro von Werra and
+                     Nazneen Rajani and
+                     Nathan Lambert
+                   },
+    title        = { StackLLaMa: An RL Fine-tuned LLaMa Model for Stack Exchange Question and Answering },
+    year         = 2023,
+    url          = { https://huggingface.co/trl-lib/llama-7b-se-rl-peft },
```
Comment (natolambert) [inline stackllama.md:284]:
```suggestion
    url          = { https://huggingface.co/blog/stackllama },
```

**Review by osanseviero — APPROVED** [review]
Thanks! 🔥

File: stackllama.md [hunk 2]
```diff
@@ -55,10 +55,10 @@ For the reward model, we will always need two answers per question to compare, a
 
 Even training the smallest LLaMA model requires an enormous amount of memory. Some quick math: in bf16, every parameter uses 2 bytes (in fp32 4 bytes) in addition to 8 bytes used, e.g., in the Adam optimizer (see the [performance docs](https://huggingface.co/docs/transformers/perf_train_gpu_one#optimizer) in Transformers for more info). So a 7B parameter model would use `(2+8)*7B=70GB` just to fit in memory and would likely need more when you compute intermediate values such as attention scores. So you couldn’t train the model even on a single 80GB A100 like that. You can use some tricks, like more efficient optimizers of half-precision training, to squeeze a bit more into memory, but you’ll run out sooner or later.
 
-Another option is to use Parameter-Efficient Fine-Tuning (PEFT) techniques, such as the [`peft`](https://github.com/huggingface/peft) library, which can perform low-rank adaption (LoRA) on a model loaded in 8-bit. 
+Another option is to use Parameter-Efficient Fine-Tuning (PEFT) techniques, such as the [`peft`](https://github.com/huggingface/peft) library, which can perform low-rank adaptation (LoRA) on a model loaded in 8-bit.
```
Comment (meg-huggingface) [inline stackllama.md:58]: Actually it's Low-Rank Adaptation.

File: stackllama.md [hunk 3]
```diff
@@ -55,10 +55,10 @@ For the reward model, we will always need two answers per question to compare, a
...
 
-Another option is to use Parameter-Efficient Fine-Tuning (PEFT) techniques, such as the [`peft`](https://github.com/huggingface/peft) library, which can perform low-rank adaption (LoRA) on a model loaded in 8-bit. 
+Another option is to use Parameter-Efficient Fine-Tuning (PEFT) techniques, such as the [`peft`](https://github.com/huggingface/peft) library, which can perform low-rank adaptation (LoRA) on a model loaded in 8-bit. 
 
 ![](https://huggingface.co/datasets/trl-internal-testing/example-images/resolve/main/blog/stackllama/lora-animated.gif)   
-*Low-Rank adaption of linear layers: extra parameters (in orange) are added next to the frozen layer (in blue), and the resulting encoded hidden states are added together with the hidden states of the frozen layer.*
+*Low-Rank adaptation of linear layers: extra parameters (in orange) are added next to the frozen layer (in blue), and the resulting encoded hidden states are added together with the hidden states of the frozen layer.*
```
Comment (meg-huggingface) [inline stackllama.md:61]:
```suggestion
*Low-Rank Adaptation of linear layers: extra parameters (in orange) are added next to the frozen layer (in blue), and the resulting encoded hidden states are added together with the hidden states of the frozen layer.*
```

File: stackllama.md (hunk 2 above)
Comment (meg-huggingface) [inline stackllama.md:58]:
```suggestion
Another option is to use Parameter-Efficient Fine-Tuning (PEFT) techniques, such as the [`peft`](https://github.com/huggingface/peft) library, which can perform Low-Rank Adaptation (LoRA) on a model loaded in 8-bit. 
```

File: stackllama.md [hunk 4]
```diff
@@ -191,7 +191,7 @@ Once more, we utilize `peft` for memory-efficient training, which offers an extr
 for epoch, batch in tqdm(enumerate(ppo_trainer.dataloader)):
     question_tensors = batch["input_ids"]
 		
-		# sample from the policy and to generate reponses
+	# sample from the policy and to generate responses
```
Comment (meg-huggingface) [inline stackllama.md:194]:
```suggestion
	# sample from the policy and generate responses
```

File: stackllama.md [hunk 5]
```diff
@@ -207,7 +207,7 @@ for epoch, batch in tqdm(enumerate(ppo_trainer.dataloader)):
 
     # Run PPO step
     stats = ppo_trainer.step(question_tensors, response_tensors, rewards)
-		# Log stats to Wandb
+	# Log stats to Wandb
```
Comment (meg-huggingface) [inline stackllama.md:210]:
```suggestion
	# Log stats to WandB
```

File: stackllama.md [hunk 6]
```diff
@@ -266,6 +266,27 @@ Once you add the evaluation step, the fun begins: you can start iterating on you
...
+@misc {beeching2023stackllama,
+    author       = { Edward Beeching and
+                     Younes Belkada and
+                     Kashif Rasul and
+                     Lewis Tunstall and
+                     Leandro von Werra and
+                     Nazneen Rajani and
+                     Nathan Lambert
+                   },
+    title        = { StackLLaMa: An RL Fine-tuned LLaMa Model for Stack Exchange Question and Answering },
```
Comment (meg-huggingface) [inline stackllama.md:282]:
```suggestion
    title        = { StackLLaMA: An RL Fine-tuned LLaMA Model for Stack Exchange Question and Answering },
```

**Review by meg-huggingface — APPROVED** [review]
Nits on nits! =)

**Pull request merged**